charts/spire/templates/server-configmap.yaml

```diff
@@ -8,13 +8,16 @@ data:
     server {
       bind_address = "0.0.0.0"
       bind_port = "8081"
-      socket_path = "{{ include "spire.sockets" . }}/registration.sock"
+      socket_path = "/run/spire/server-sockets/registration.sock"
       trust_domain = {{ .Values.spire.trustDomain | quote }}
       data_dir = "/run/spire/data"
       log_level = "{{ .Values.spire.server.logLevel }}"
       #AWS requires the use of RSA.  EC cryptography is not supported
       ca_key_type = "rsa-2048"
 
+      {{- if eq (.Values.oidc.enabled | toString) "true" }}
+      jwt_issuer = "{{ .Values.oidc.domain }}"
```

Comment:
The issuer should be retrieved from the `domains` setting now. Maybe we should make it by convention to take the first.